BGAnimations/ScreenSystemLayer overlay.lua

```diff
@@ -208,12 +212,18 @@ t[#t+1] = LoadFont("Common Footer")..{
 	end
 }
 
+function updateTimestamp(af)
+	timestamp:playcommand("Refresh")

```

Comment:
Just for safety, we should wrap this with `if timestamp then`

Reply:
Done!